### PR TITLE
Make models translatable and add Russian translation

### PR DESCRIPTION
**Pull request by Real-Gecko**

### CODE REVIEW

File: easyaudit/models.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```
Comment (jheld) [inline easyaudit/models.py:4]: I think django is deprecating this in favor of `gettext_lazy`. Can you look into it before merge?

**Review by jheld — CHANGES_REQUESTED** [review]
Just a small thing about a django import, looks good otherwise, thank you!

File: easyaudit/models.py (hunk 1 above)
Comment (Real-Gecko) [reply easyaudit/models.py]: Yes you're right `ugettext_lazy` is deprecated and is alias for `gettext_lazy`, commited changes.

**Review by jheld — APPROVED** [review]

**Comment by jheld** [comment]
Probably won't cut a release just yet (depends on your other PR and any others), but likely will within the next 2 or 3 weeks. Please ping me if I haven't.

**Comment by Real-Gecko** [comment]
There's one more PR worth checking #163

**Pull request merged**